dev/com.ibm.ws.jaxrs.2.1_fat_extended/fat/src/com/ibm/ws/jaxrs21/fat/extended/SubResourceTest.java

```diff
@@ -15,20 +15,27 @@
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.runner.RunWith;
 
 import com.ibm.websphere.simplicity.ShrinkHelper;
 
 import componenttest.annotation.Server;
 import componenttest.annotation.TestServlet;
 import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.RepeatTests;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.FATServletClient;
 import jaxrs21.fat.subresource.SubResourceTestServlet;
 
 @RunWith(FATRunner.class)
 public class SubResourceTest extends FATServletClient {
 
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
+                                             .andWith(FeatureReplacementAction.EE7_FEATURES());
+                    
```

Comment:
It would be convenient to add ```.andWith(FeatureReplacementAction.EE7_FEATURES().withID("JAXRS-2.0")```  so that we can tell if the test is running in jaxrs-2.0 mode